loopd/utils.go

```diff
@@ -31,26 +31,40 @@ func getClient(cfg *Config, lnd *lndclient.LndServices) (*loop.Client,
 		MaxPaymentRetries:   cfg.MaxPaymentRetries,
 	}
 
-	// Now that we know where the database will live, we'll go ahead and
-	// open up the default implementation of it.
+	swapClient, cleanUp, err := loop.NewClient(
+		cfg.DataDir, swapDb, clientConfig,
+	)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return swapClient, cleanUp, nil
+}
+
+func openDatabase(cfg *Config, network *chaincfg.Params) (
```

Comment:
nit: instead of `network` I'd suggest calling this `chainParams` or `params` (we use this naming elsewhere in the codebase).